src/main/java/ch/njol/skript/expressions/ExprDyed.java

```diff
@@ -0,0 +1,133 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.Color;
+import ch.njol.skript.util.ColorRGB;
+import ch.njol.util.Kleenean;
+import org.bukkit.Material;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.inventory.meta.LeatherArmorMeta;
+import org.bukkit.inventory.meta.MapMeta;
+import org.bukkit.inventory.meta.PotionMeta;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.Locale;
+import java.util.regex.Matcher;
+
+@Name("Dyed")
+@Description("An expression to return colored items.")
+@Examples({
+		"give player leather chestplate dyed red",
+		"give player potion of invisibility dyed rgb 200, 70, 88",
+		"give player filled map colored rgb(20, 60, 70)",
+		"give player wool painted red"
```

Comment:
```suggestion
	"give player leather chestplate dyed red",
	"give player potion of invisibility dyed rgb 200, 70, 88",
	"give player filled map colored rgb(20, 60, 70)",
	"give player wool painted red"
```
I think one line is okay for these annotations